homeassistant/components/mazda/device_tracker.py

```diff
@@ -3,17 +3,18 @@
 from homeassistant.components.device_tracker.config_entry import TrackerEntity
 
 from . import MazdaEntity
-from .const import DATA_COORDINATOR, DOMAIN
+from .const import DATA_CLIENT, DATA_COORDINATOR, DOMAIN
 
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the device tracker platform."""
+    client = hass.data[DOMAIN][config_entry.entry_id][DATA_CLIENT]
```

Comment:
An alternate option to avoid the refactor would be to store the client as a property of the coordinator in a subclass

Example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/roku/__init__.py#L80

It fine either way, but this might be easier to maintain.

Reply:
I think I'd like to keep it as is, if that's OK. But thanks for letting me know about this!